src/components/30-organisms/modal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axa-ch/modal",
-  "version": "4.0.9",
+  "version": "4.0.10",
```

Comment:
Please do **not** manually increment package.json version numbers in the PL.

This is taken care of by `lerna version` as part of the CLI command chain in `npm run release`, that's where you choose what kind of change this is and lerna proposes a semantically correct (in the sense of semver) new version.